### PR TITLE
Test/front end tests

### PR DESCRIPTION
**Pull request by davidlag0**
## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [x] Other (please describe): Tests

## What is the current behavior?

Issue Number: #135 

## What is the new behavior?

Snapshot tests for components NetworkManagement, Bar and HomeLoggedIn

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -25,9 +25,9 @@
   "devDependencies": {
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "12.1.5",
+    "@testing-library/user-event": "^14.4.3",
```
Comment (dec0dOS) [inline frontend/package.json:29]:
```
❯ yarn install
➤ YN0000: ┌ Resolution step
➤ YN0002: │ frontend@workspace:frontend doesn't provide @testing-library/dom (p5137c), requested by @testing-library/user-event
➤ YN0002: │ frontend@workspace:frontend doesn't provide postcss (p3d569), requested by jest-transform-css
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed
```
Additional dependencies should be added

File: frontend/package.json [hunk 2]
```diff
@@ -25,9 +25,9 @@
...
     "axios-mock-adapter": "^1.21.2",
-    "jest": "^29.3.1",
-    "jest-environment-jsdom": "^29.3.1",
+    "jest": "26.6.0",
```
Comment (dec0dOS) [inline frontend/package.json:31]: Could you explain please why you're using the older jest version?

**Review by dec0dOS — COMMENTED** [review]
Please check the dependencies comments. In the code seems to be everything if fine. Thank you for your work, @davidlag0!

File: frontend/package.json (hunk 2 above)
Comment (davidlag0) [reply frontend/package.json:31]: Yes, the current version of react-scripts is dependent on Jest 26.6.0. If we upgrade to a newer version of Jest, it breaks things so I'm using that older version of Jest until we can start upgrading some other packages such as MUI and React.

**Comment by davidlag0** [comment]
@dec0dOS Good catch for the missing dependencies. Should be resolved now.

**Pull request merged**